joint_state_publisher/joint_state_publisher.py

```diff
@@ -483,7 +482,19 @@ def main(input_args=None):
     with open(parsed_args.urdf_file, 'r') as infp:
         urdf = infp.read()
 
-    node = rclpy.create_node('joint_state_publisher')
+    initial_parameters=[
+        rclpy.parameter.Parameter('num_rows', rclpy.parameter.Parameter.Type.INTEGER, 0),
+        rclpy.parameter.Parameter('publish_default_efforts', rclpy.parameter.Parameter.Type.BOOL, False),
+        rclpy.parameter.Parameter('publish_default_positions', rclpy.parameter.Parameter.Type.BOOL, True),
+        rclpy.parameter.Parameter('publish_default_velocities', rclpy.parameter.Parameter.Type.BOOL, False),
+        rclpy.parameter.Parameter('rate', rclpy.parameter.Parameter.Type.INTEGER, 10),  # 10hz
+        rclpy.parameter.Parameter('source_list', rclpy.parameter.Parameter.Type.STRING_ARRAY, []),
+        rclpy.parameter.Parameter('use_gui', rclpy.parameter.Parameter.Type.BOOL, False),
+        rclpy.parameter.Parameter('use_mimic_tags', rclpy.parameter.Parameter.Type.BOOL, True),
+        rclpy.parameter.Parameter('use_smallest_joint_limits', rclpy.parameter.Parameter.Type.BOOL, True),
+    ]
+
+    node = rclpy.create_node('joint_state_publisher', initial_parameters=initial_parameters)
```

Comment:
Looks like the `initial_parameters` argument was renamed to `parameter_overrides` in the Dashing release.

The values set here seem to override the parameters I'm setting in my launch file, which probably isn't desirable behavior (i.e. setting `use_gui` to `true` in the launch file has no effect unless I comment out line 492).